### DISCUSSION TITLE
closed orbit amplitude in 6D with radiation ON

### DISCUSSION THREAD

**Discussion by lnadolski** [atcollab/at#551] [Q&A]
Dear All,

Computing the closed orbit in 6D with radiation ON, I am puzzled by the amplitude in the horizontal plane which can reach 100µm.

I take the example of esrf lattice in the following plot

atplot(atenable_6d(esrf,'auto'), @plClosedOrbit)
![image](https://user-images.githubusercontent.com/3135838/212298070-969f57b4-a268-4881-93dc-3dce2f7c1ab0.png)

Does it make sense for you to you?
The averaged energy loss per turn is 8.0773e-04.

Laurent.

**Comment by simoneliuzzo** [comment]
For me this plot 6D, radiation ON, should give the "saw tooth", horizontal orbit slowly oscillating to negative values, and recovering at the RF cavities. This is order of micrometers for esrf. I tried the code that you propose and also `atplot(atsetcavity(esrf,8e6,1,992),@plClosedOrbit)` the result is identical to the figure that you show.  
I can not produe the figure for ESRF, evidently, but for EBS it looks like this:
![Screenshot 2023-01-18 at 13 36 34](https://user-images.githubusercontent.com/30342742/213173095-7ba22ee5-8edf-4443-b7d0-d61d26905466.png)
I think @swhite2401 and @lfarv are needed to explain this.

**Comment by swhite2401** [comment]
Hello @lnadolski , I confirm your result for the esrf lattice from this AT `machine_data` folder. However, looking at the HMBA lattice I get the following:

![image](https://user-images.githubusercontent.com/10014116/213174320-e1d036b3-a9a4-4950-a73e-f35ffe2adb9f.png)

This is what is expected, I do not know this `esrf.m` lattice or how it was generated, maybe it has a problem?

**Comment by swhite2401** [comment]
Ah @simoneliuzzo has the same answer, well synchronized! @simoneliuzzo do you know of any problem with this lattice? Wasn't there an issue with the total bending angle?

**Comment by lnadolski** [comment]
This is strange.
I got similar results to esrf.m for SOLEIL lattice 
I got similar results to hmba for SOLEIL II (see plot below)
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/3135838/213310263-e8c0d018-a30f-47e4-9c42-67492dec6596.png">

**Comment by swhite2401** [comment]
This is in fact strange... are the passmethods the same in all these lattices?

**Comment by simoneliuzzo** [comment]
Dear colleagues,

After looking more into this issue, I think it is somehow a feature of multi bend lattices to have a clearly recognizable sawthoot effect.

We may expect larger 6D orbit for DBA/TBA lattices compared to MBA. For the upgrades dispersion is smaller as well as energy loss, so the orbit due to delta-vs-s is smaller (max(delta)*max(disp) ~ 8um for EBS, ~140 um for ESRF-DBA). 

I have done several tests and among them I fixed the geometry of the esrf.m lattice that is slightly wrong as suggested by @carmignani. (I attach here a file with corrected angles in all dipoles to close the geometry). 
[esrf_closed_geometry_padpolynomAB_disabled6d.m.zip](https://github.com/atcollab/at/files/12026527/esrf_closed_geometry_padpolynomAB_disabled6d.m.zip)

For the 6D orbit I do not see changes, and I am convincing myself that it is the correct orbit. 
![Screenshot 2023-07-12 at 10 38 52](https://github.com/atcollab/at/assets/30342742/148abe30-40bf-45cb-91d7-d5483a03aa1d)

I try to place 1 cavity in each cell of the DBA esrf lattice. The expectation is that the energy changes are smaller, thus we should see smaller over all 6D orbit and a zig/zag, dispersive pattern with positive orbit in the first half (higher energy) of the cell and negative in the second half (lower energy).
![Screenshot 2023-07-12 at 11 42 29](https://github.com/atcollab/at/assets/30342742/0b2d9a7a-46eb-471b-a971-755134478eb8)

**Reply by lnadolski** [reply]
Thank @simoneliuzzo for continuing the investigations.

So this is just the impact of the energy loss along the orbit within a turn.
Since the mcf is smaller, the impact is much less for 4GLS lattices.
